Sources/Core/Tracker/TrackerControllerImpl.swift

```diff
@@ -73,6 +73,14 @@ class TrackerControllerImpl: Controller, TrackerController {
     func track(_ event: Event) -> UUID? {
         return tracker.track(event)
     }
+    
+    func decorateLink(_ url: URL) -> URL? {
+        return tracker.decorateLink(url)
+    }
+    
+    func decorateLink(_ url: URL, extendedParameters: CrossDeviceParameterConfiguration) -> URL? {
+        return tracker.decorateLink(url, extendedParameters: extendedParameters)
```

Comment:
What do you think about implementing the `decorateLink` function here instead of the `Tracker` class? The `Tracker` class should be the core functionality of the tracker (although currently it's a bit bloated) so I am a bit worried about adding more to the pile.